tests/Unit/Time/Actions/Test_SelfStartActions.cpp

```diff
@@ -205,33 +250,36 @@ using not_self_start_action = std::negation<std::disjunction<
 // Fail a REQUIRE if any action not passing the Whitelist metalambda
 // is run first (as that would often lead to an infinite loop).
 // Returns true if the last action jumped.
-template <typename Stop, typename Whitelist, bool HasPrimitives>
+template <typename Stop, typename Whitelist, bool MultipleHistories = false,
+          bool HasPrimitives>
```

Comment:
Default isn't useful here.  If `HasPrimitives` can be deduced, then so can `MultipleHistories`.